dll/win32/devmgr/lang/pt-PT.rc

```diff
@@ -7,10 +7,10 @@ BEGIN
     IDS_NAME "Nome"
     IDS_TYPE "Tipo"
     IDS_MANUFACTURER "Fabricante: %1"
-    IDS_LOCATION "Local: %1"
-    IDS_STATUS "Estado do dispositivo: %1"
+    IDS_LOCATION "Localização: %1"
+    IDS_STATUS "Estado do\ndispositivo: %1"
```

Comment:
Are you sure about using "\n" line break here? Other translations does not seem to do that:

![image](https://user-images.githubusercontent.com/578406/149519155-2de2f225-fb5f-4385-816f-8d73943639fd.png)

Reply:
![ReactOS](https://user-images.githubusercontent.com/49964946/149578643-404f4e7a-00a3-4566-baa7-37d6be9f56dc.png)
![ReactOS2](https://user-images.githubusercontent.com/49964946/149578658-5add00bb-0fb3-4d9c-b2a0-a6c9d6560b6c.png)

Improved visual arrangement

Reply:
The "before" looks better for me. It's also the reason why this label supports word wrapping.

Reply:
OK. I accept the majority opinion. I like the altered version as it is more aligned .

Reply:
I'm pretty sure this is how it's done in Windows, might be a good idea to double check.

Reply:
Maybe I have a better solution for both.
"Estado disp.." Like in Win7 pt-PT

Reply:
Maybe the best option would be to resize the underlying dialog box itself.

Reply:
I like that solution.

Reply:
Hmm... I actually tried to resize the `IDD_HARDWARE` dialog box, and it seems it's not possible, as the property sheet is then resized automatically.

While checking, I looked up the dialog in Restorator, and noticed there's also problem with "fix conflicts" button, which is usually hidden. Here's the fix for it:
```
PUSHBUTTON "&Resolução de problemas...", IDC_TROUBLESHOOT, 42, 140, 97, 14, BS_PUSHBUTTON | WS_CHILD | WS_DISABLED | WS_TABSTOP
```